### DISCUSSION TITLE
Nextjs 13.5.3 error connecting to Planetscale db

### DISCUSSION THREAD

**Discussion by Nithur-M** [vercel/next.js#56196] [Help]
### Summary

I am facing this error after upgrading to Nextjs 13.5.3:
`
Failed to set fetch cache https://us-east.connect.psdb.cloud/psdb.v1alpha1.Database/Execute TypeError: fetch failed
--
10:15:26.678 | at Object.fetch (/vercel/path0/node_modules/next/dist/compiled/undici/index.js:1:26672)
10:15:26.678 | at async globalThis.fetch (/vercel/path0/node_modules/next/dist/server/lib/patch-fetch.js:199:16)
10:15:26.678 | at async invokeRequest (/vercel/path0/node_modules/next/dist/server/lib/server-ipc/invoke-request.js:17:12)
10:15:26.678 | at async invokeIpcMethod (/vercel/path0/node_modules/next/dist/server/lib/server-ipc/request-utils.js:45:21)
10:15:26.679 | at async /vercel/path0/node_modules/next/dist/server/lib/patch-fetch.js:396:29 {
10:15:26.679 | cause: SocketError: other side closed
10:15:26.679 | at Socket.onSocketEnd (/vercel/path0/node_modules/next/dist/compiled/undici/index.js:1:73361)
10:15:26.679 | at Socket.emit (node:events:525:35)
10:15:26.679 | at endReadableNT (node:internal/streams/readable:1358:12)
10:15:26.679 | at processTicksAndRejections (node:internal/process/task_queues:83:21) {
10:15:26.680 | code: 'UND_ERR_SOCKET',
10:15:26.680 | socket: {
10:15:26.680 | localAddress: '127.0.0.1',
10:15:26.680 | localPort: 56152,
10:15:26.680 | remoteAddress: '127.0.0.1',
10:15:26.681 | remotePort: 39405,
10:15:26.681 | remoteFamily: 'IPv4',
10:15:26.681 | timeout: undefined,
10:15:26.681 | bytesWritten: 19995,
10:15:26.681 | bytesRead: 18061
10:15:26.682 | }
10:15:26.682 | }
`
Does anyone have any idea what's causing this error? Thanks.

**Comment by leerob** [comment]
Please include a reproduction or public GitHub issue and open a new Discussion so folks can help out! It's best if the example is extremely minimal. Thank you.
